dom.bs

```diff
@@ -5135,8 +5171,7 @@ method steps are to return the <a>list of elements with class names <var>classNa
   <a for=/>node</a> whose
   <a for=ProcessingInstruction>target</a> is <var>target</var> and
   <a for=CharacterData>data</a> is <var>data</var>.
-  If <var>target</var> does not match the
-  <code><a type>Name</a></code> production an
+  If <var>target</var> is not a <a>valid basic name</a> an
```

Comment:
This changes the validation for `ProcessingInstruction` in a domintro box. That seems like an oversight.